src/Command/FrontendCommands.php

```diff
@@ -150,8 +151,15 @@ private function frontendLink()
     * "extra" section of DKAN's composer.json. If you want to specify a
     * different tag or branch, or different repo entirely, run "dktl
     * frontend:get" first and specify the --ref and/or --url options.
+    *
+    * @param array $opts
+    *   Options array.
+    * @option theme
+    *   Composer requirement for theme to download and enable.
+    *   Pass --no-theme to skip theme installation. Defaults to
+    *   "getdkan/dkan_js_frontend_bartik".
     */
-    public function frontendInstall()
+    public function frontendInstall($opts = ['theme' => true])
     {
```

Comment:
```suggestion
    public function frontendInstall($opts = ['no-theme' => false, 'theme' => self::FRONTEND_THEME])
    {
        $theme = $opts['no-theme'] ? NULL : $opts['theme'];
```
I did some research into this, and it looks like you can't mix argument types in Robo. I think the only way to achieve the desired functionality is to include the "--no" argument as a second option.

Reply:
@clayliddell when you set an opt to `true` by default it automatically provides a `--no-[opt]`, check the help text for the command as it is and you should see it. I agree the whole thing is a little awkward, the thing is I doubt we will even use this option any time soon and maybe I shouldn't have included it but forcing that particular theme seemed off so this seemed to be the least weird way to deal with it. Can you take another look an lmk what you think?

Reply:
If you set an opt to `true`, it will expect a boolean, and won't take a `string`. It looks like you expect the theme option to be able to take a string (so that users can specify custom themes). Is my understanding incorrect?

Reply:
switching to boolean only